Source/DafnyCore/Dafny.atg

```diff
@@ -952,19 +981,25 @@ Dafny
      DefaultModuleDecl defaultModule = (DefaultModuleDecl)((LiteralModuleDecl)theModule).ModuleDef;
      // theModule should be a DefaultModuleDecl (actually, the singular DefaultModuleDecl)
      Contract.Assert(defaultModule != null);
-  .)
-  { "include" stringToken               (. {
-                                             string parsedFile = scanner.FullFilename;
-                                             bool isVerbatimString;
-                                             string includedFile = Util.RemoveParsedStringQuotes(t.val, out isVerbatimString);
-                                             includedFile = Util.RemoveEscaping(includedFile, isVerbatimString);
-                                             if (!Path.IsPathRooted(includedFile)) {
-                                               string basePath = Path.GetDirectoryName(parsedFile);
-                                               includedFile = Path.Combine(basePath, includedFile);
-                                             }
-                                             defaultModule.Includes.Add(new Include(t, parsedFile, includedFile, theCompileThisFile));
-                                           }
-                                        .)
+     var ownedTokens = new List<IToken>();
+  .)
+  { "include"                 (. ownedTokens.Add(t); .)
```

Comment:
A general comment about this file - It seems unfortunate that we need to manually track owned tokens in the grammar, but I can understand that assigning ownership post-parse might not be any cleaner. I'm concerned that later changes/additions to the grammar might introduce missing/incorrect `ownedTokens`. Do we at least have a test to check that every token is accounted for in some `ownedTokens`?